src/run.rs

```diff
@@ -50,6 +50,7 @@ pub(crate) async fn pull_and_run(
     let policy_evaluator = PolicyEvaluator::from_file(
         policy_id,
         policy_path,
+        PolicyExecutionMode::Opa,
```

Comment:
This is here just to try if you want. We have to discuss whether we want to add a flag besides the metadata that we are going to read in order to use one method of evaluation or another.